app/views/layouts/application.html.erb

```diff
@@ -15,6 +15,9 @@
     <link rel="manifest" href="/manifest.json">
     <link rel="apple-touch-icon" href="<%= image_path('apple-touch-icon.png') %>">
     <link rel="icon" href="<%= image_path('favicon.png') %>">
+    <%= javascript_tag do %>
+      window.pricesURL = '<%= ENV.fetch('COINFI_PRICES_URL', '') %>';
+    <% end %>
```

Comment:
@hanchang I am really not sure this is the best way to achieve this, and am open to other suggestions. I was thinking data attributes possibly, but this could be used in a number of places potentially.

However, if we're moving to react, I'm not sure it will be an issue for long.

Reply:
Yup that works for me, thanks for checking!